scripts/lib/CIME/test_scheduler.py

```diff
@@ -340,7 +342,7 @@ def __init__(
                     test_baseline = os.path.join(full_baseline_dir, test_name)
                     if os.path.isdir(test_baseline):
                         existing_baselines.append(test_baseline)
-                        if allow_baseline_overwrite:
+                        if allow_baseline_overwrite and retry == "None":
```

Comment:
Should this really be `retry == "None"` or should it be `retry is None`?

Reply:
Yes is should be.  I just pushed the change.

Reply:
That is it should be retry is None.